index.js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+
+const path = require('path');
```

Comment:
Maybe you could import these variables from utils.js so you don't repeat this lines.